zulipterminal/cli/run.py

```diff
@@ -105,7 +105,7 @@ def fetch_zuliprc(zuliprc_path: str) -> None:
 
```

Comment:
You can shorten/simplify this commit title now :) (run: Refactor parse_zuliprc...)

Reply:
Fixed